OTAnalytics/plugin_ui/customtkinter_gui/helpers.py

```diff
@@ -13,3 +27,24 @@ def get_widget_position(widget: Widget) -> tuple[int, int]:
     x = widget.winfo_rootx()
     y = widget.winfo_rooty()
     return x, y
+
+
+def coordinate_is_on_widget(coordinate: tuple[int, int], widget: Widget) -> bool:
+    """Returns True, if the coordinate is within the boundaries of an existing
+        tkinter (or customtkinter) widget.
+
+    Args:
+        coordinate (tuple[int, int]): Coordinate
```

Comment:
Should we add a hint, that the coordinates should be relative to the widget position?

Otherwise one might use display coordinates.